dist/js/incito-publication.js

```diff
@@ -4,6 +4,9 @@
             graphUrl: 'https://graph.service-staging.shopgun.com'
         });
     }
+    SGN.config.set({
```

Comment:
Perhaps you could remove all the staging configs as we don't really want them here.